Patch by Sergei Golovan removes unnecessary call to ssl:start() because
ssl module isn't used in ejabberd anymore (LDAP module will start ssl itself).

Index: ejabberd-2.0.5/src/ejabberd_app.erl
===================================================================
--- ejabberd-2.0.5.orig/src/ejabberd_app.erl	2009-04-01 17:23:52.000000000 +0200
+++ ejabberd-2.0.5/src/ejabberd_app.erl	2009-04-05 22:51:20.000000000 +0200
@@ -44,7 +44,7 @@
     randoms:start(),
     db_init(),
     sha:start(),
-    catch ssl:start(),
+%%  catch ssl:start(),
     stringprep_sup:start_link(),
     start(),
     translate:start(),
